### PR TITLE
Add video service

### PR DESCRIPTION
**Pull request by supasate**
Add video service to upload a video via `POST /videos` for #56 

Note: 
- Will refactor the video and photo service to reuse the common component.
- Need to add test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "feathers-configuration": "^0.2.3",
     "feathers-errors": "^2.2.0",
     "feathers-hooks": "^1.5.4",
-    "feathers-mongoose": "^3.5.0",
+    "feathers-mongoose": "^3.5.1",
```
Comment (parnurzeal) [inline package.json:33]: What is the difference between 3.5.0 and 3.5.1?

File: src/middleware/attach-file-to-feathers.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+// Middle to attach file from multer (uploader) to the req object
```
Comment (parnurzeal) [inline src/middleware/attach-file-to-feathers.js]: s/Middle/Middleware/

File: src/services/index.js [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 
+// Services
+const authentication = require('./authentication');
+const user = require('./user');
+const pin = require('./pin');
+const photo = require('./photo');
+const video = require('./video');
+const searchnearby = require('./searchnearby');
+const app3rd = require('./app3rd');
+
```
Comment (parnurzeal) [inline src/services/index.js:14]: Great! Thanks for reordering it :)
nit: reorder alphabetically on the Services models.

File: src/services/video/index.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const Promise = require('bluebird');
+const errors = require('feathers-errors');
+const request = require('superagent');
```
Comment (parnurzeal) [inline src/services/video/index.js]: reorder alphabetically?

File: src/services/video/index.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+      reqFile.cloudStoragePublicUrl = publicUrl;
+
+      return resolve(reqFile);
+    });
+
+    stream.end(reqFile.buffer);
+  });
+}
+
+function getMetadataFromUrl(url) {
```
Comment (parnurzeal) [inline src/services/video/index.js]: where is this used for?

Comment (supasate) [inline src/services/video/index.js]: Forgot to delete. Thx!

File: package.json (hunk 1 above)
Comment (supasate) [inline package.json:33]: Don't check yet. It was replaced automatically by feathers service generator.

File: src/services/index.js (hunk 3 above)
Comment (supasate) [inline src/services/index.js:14]: Ok

File: src/services/video/index.js (hunk 4 above)
Comment (supasate) [inline src/services/video/index.js]: Ok

**Comment by supasate** [comment]
Already fixed based on all comments @parnurzea.l

File: package.json (hunk 1 above)
Comment (supasate) [inline package.json:33]: They do not write changelog about 3.5.1. Maybe some minor fixes.

Comment (parnurzeal) [inline package.json:33]: Ok.

**Pull request merged**